docs/en/setup/plugins/fetcher_prometheus-metrics-fetcher.md

```diff
@@ -1,13 +1,34 @@
 # Fetcher/prometheus-metrics-fetcher
+
```

Comment:
@mrproliu This should be generated by the comments?

Reply:
Yes, I used `make gen-docs` instead.